apps/bare-expo/e2e/TestSuite-test.web.js

```diff
@@ -0,0 +1,55 @@
+/* global page */
+import { setDefaultOptions } from 'expect-puppeteer';
+
+import config from '../jest-puppeteer.config';
+import { expectResults } from './utils/report';
+
+const sleepAsync = t => new Promise(res => setTimeout(res, t));
+
+let TESTS = [
+  'Basic',
+  'Constants',
+  'Crypto',
+  //   'Haptics',
+  'Localization',
+  //   'SecureStore',
+  'Random',
+  //   'Permissions',
+];
+
+const MIN_TIME = 50000;
+
+setDefaultOptions({
+  timeout: MIN_TIME * 1.5,
+});
+
+function matchID(id, ...props) {
+  return expect(page).toMatchElement(`div[data-testid="${id}"]`, ...props);
+}
+
+describe('test-suite', () => {
+  TESTS.map(testName => {
+    it(
+      `passes ${testName}`,
+      async () => {
+        /// Pause the timeout
+        // await jestPuppeteer.debug();
+
+        await page.goto(`${config.url}/test-suite/select/${testName}`);
+        await sleepAsync(100);
```

Comment:
instead of sleeping, wait for the thing we actually care about (eg domcontentloaded)